JASP-Engine/JASP/R/multinomialtestbayesian.R

```diff
@@ -199,13 +199,13 @@ MultinomialTestBayesian <- function(jaspResults, dataset, options, ...) {
   if(bayesianAnalysis){
     ciRequested   <- options$credibleInterval
     ciInterval    <- options$credibleIntervalInterval
-    ciColumnName  <- gettext("Credible Interval")
+    ciType  <- gettext("Credible")
     tableFootnote <- gettext("Credible intervals are based on independent binomial distributions with flat priors.")
     descriptivesTable$dependOn(c("countProp", "descriptives", "credibleIntervalInterval"))
   } else {
     ciRequested   <- options$confidenceInterval
     ciInterval    <- options$confidenceIntervalInterval 
-    ciColumnName  <- gettext("Confidence Interval")
+    ciType  <- gettext("Confidence")
```

Comment:
Why do we assign the word Confidence to the ciType ? Translation is so more unclear